src/h5p.vue

```diff
@@ -125,6 +125,9 @@ export default {
       ),
       ...this.integration
     }
+    if (h5p && h5p.title) {
+      h5pIntegration.contents['cid-default'].metadata = h5p
+    }
```

Comment:
I don't think we need the `if` condition at all. We should just do something like this a bit further up, where we create the whole `contents` object:

```
metadata: {
  license: h5p.license,
  defaultLanguage: h5p.defaultLanguage,
  title: h5p.title
}
```

This will avoid setting the full h5p objects, which contains a lot more as metadata. And h5p.org only sets those 3 keys, as mentioned in the other comment.

Reply:
I agree with you maybe the if is not needed, but since h5p object was inside a try catch block, I wanted to be sure it is properly defined (bad development practice on my side)

Reply:
Right. There is a `return` in the `catch`, though - so `h5p` will always be defined at this stage.